pandas/io/parsers.py

```diff
@@ -738,10 +740,10 @@ def _make_engine(self, engine='c'):
         if engine == 'c':
             self._engine = CParserWrapper(self.f, **self.options)
         else:
-            if engine == 'python':
-                klass = PythonParser
-            elif engine == 'python-fwf':
+            if engine == 'python-fwf':
                 klass = FixedWidthFieldParser
+            else:  #default to engine == 'python':
```

Comment:
For the use case that engine is neither `'python'` nor `'python-fwf'`.

Is it possible for this to happen?